openstack/clustering/v1/policies/requests.go

```diff
@@ -83,3 +85,13 @@ func Create(client *gophercloud.ServiceClient, opts CreateOpts) (r CreateResult)
 	})
 	return
 }
+
+// Create makes a request against the API to delete a policy
+func Delete(client *gophercloud.ServiceClient, policyID string) (r DeleteResult) {
```

Comment:
I left the Delete function as is because the test case `th.AssertEquals(t, PolicyDeleteRequestID, actual.RequestID)`  kept failing when I tried it the simplified way.

Reply:
That sounds good. I'll have to double-check the object storage implementation when I get a chance.